hrm-domain/lambdas/search-index-consumer/messagesToPayloads.ts

```diff
@@ -145,7 +145,7 @@ const generatePayloadFromContact = (
           ...(ps[HRM_CONTACTS_INDEX_TYPE] ?? []),
           {
             ...m,
-            documentId: m.message.contact.id,
+            documentId: parseInt(m.message.contact.id),
```

Comment:
In `handleIndexPayload` (`hrm-domain/lambdas/search-index-consumer/payloadToIndex.ts`) we do `id: documentId.toString()` :stuck_out_tongue:. Maybe change the type signature of `handleIndexPayload` instead?

Reply:
Was trying to draw a line for the refactor somewhere, maybe it wasn't the best place